plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-downloads.php

```diff
@@ -20,27 +18,35 @@ class WC_Meta_Box_Order_Downloads {
 	/**
 	 * Output the metabox.
 	 *
-	 * @param WP_Post $post
+	 * @param WC_Order|WP_Post $post_or_order Post or order object.
 	 */
-	public static function output( $post ) {
+	public static function output( $post_or_order ) {
```

Comment:
This is a potentially backwards incompatible change (could lead to an unknown named parameter error). Wanted to flag, since this is a pre-existing and public method, but it does feel like a pretty low risk change overall.